nmpc_controller/src/quad_nlp.cpp

```diff
@@ -299,7 +305,7 @@ bool quadNLP::get_starting_point(
    if (init_z)
    {
       Eigen::Map<Eigen::MatrixXd> z_L_matrix(z_L, n, 1);
-      Eigen::Map<Eigen::MatrixXd> z_U_matrix(z_L, n, 1);
+      Eigen::Map<Eigen::MatrixXd> z_U_matrix(z_U, n, 1);
```

Comment:
Was this the bug that was causing the linear solver to fail?

Reply:
Yes, here and the finalize_solution() below. I accidentally recorded Z_L for Z_U.